index.html

```diff
@@ -136,6 +136,164 @@ <h2>
         sensing, depth sensing, video analysis, gesture recognition, and so on.
       </p>
     </section>
+    <section>
+      <h2>
+        Model
+      </h2>
+      <p>
+        A <dfn>gamepad</dfn> is a collection of input controls (buttons,
```

Comment:
```suggestion
        A <dfn>gamepad</dfn> is a collection of <dfn>input controls</dfn> (buttons,
```
Maybe we should also add a definition for "input controls"?

Reply:
Yes, let's add definitions for both input controls and output controls.